image_provisioner/playbooks/roles/clone-repos/tasks/main.yaml

```diff
@@ -29,13 +29,22 @@
     path: /root/openshift-ansible 
   register: openshift_ansible
 
+- name: get the openshift version
+  shell: echo "{{ openshift_version }}" |  awk -F '.' '{ print $1"."$2 }'
```

Comment:
Instead of doing a shell command, we should use Ansible to split the string:  
`openshift_version.split('.')